obp-commons/src/main/scala/com/openbankproject/commons/dto/JsonsTransfer.scala

```diff
@@ -155,6 +155,12 @@ case class OutBoundMakePaymentv210(outboundAdapterCallContext: OutboundAdapterCa
 
 case class InBoundMakePaymentv210(inboundAdapterCallContext: InboundAdapterCallContext, status: Status, data: TransactionId) extends InBoundTrait[TransactionId]
 
+case class OutBoundMakePaymentV400(outboundAdapterCallContext: OutboundAdapterCallContext,
+                                   transactionRequest: TransactionRequest,
+                                   reasons: Option[List[TransactionRequestReason]]) extends TopicTrait
+
+case class InBoundMakePaymentV400(inboundAdapterCallContext: InboundAdapterCallContext, status: Status, data: TransactionId) extends InBoundTrait[TransactionId]
```

Comment:
which `status` is it ?
and we maybe should rename the `data` field to `transactionId`